# This message is used to collect the information from all clamps found in a frame in one array.

Header header
Clamp[] clamps
